app/assets/web-dev-iframe.html

```diff
@@ -10,16 +10,26 @@
 
     <script src="https://cdn.jsdelivr.net/lodash/4.13.1/lodash.min.js" integrity="sha256-8SeyqJ7ZAZx8WnIgP/bgK6LGIjKjhojNPHSMV/fo29Y=" crossorigin="anonymous"></script>
 
+    <!-- TODO: Does jquery font loading affect china customers? -->
```

Comment:
Can you elaborate on this a bit?  Not quite sure what this means.  What does it mean to effect china customers?

Reply:
It appears in the network tab that there might be a font being loaded by this script. This font load would fail in china.
I can update the comment.

Reply:
I chose to delete this comment as it may be a red herring. We'll better know if there's a problem if China reports an issue.